index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Israel CHIZUNGU Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <section>
+        <!--  toolbar starts -->
+        <header>
+            <a href="#" class="logo">My Logo</a>
+            <a href="#"><img src="img/hamburger.png" alt="humburger icon" class="hamburger"></a>
+            
+
+        </header>
+        <!--  toolbar ends -->
+
+
+        <!--  headline starts -->
+        <main>
+            <h2 class="intro-text">I’m Mariane
+                Glad to see you!</h2>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <a href="#" class="subtitle">LET’S CONNECT</a>
+            <div class="links-group">
+                <a href="#" class="link"><img src="img/Vectortwitter.svg" alt="twitter" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Linkedin iconlinkedin.svg" alt="Linkedin" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Vectorm.svg" alt="M" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Vectorgithub.svg" alt="github" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Vectorp-l.svg" alt="Peace & Love" class="img-link"></a>
+            </div>
```

Comment:
- [ ] Kindly make sure, the list of **icons** uses a `<ul>` tag to group them(not `<div> tag)`, and `<li>` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. **i.e.:**
```HTML
  <ul>
    <li>
      <a href="#" class="link"><img src="img/Vectortwitter.svg" alt="twitter" class="img-link"></a>
    </li>
    <li>
      <a href="#" class="link"><img src="img/Vectortwitter.svg" alt="twitter" class="img-link"></a>
    </li>
    ...
  </ul>
```